src/styles/components/_doc-sidebar.scss

```diff
@@ -147,6 +159,8 @@ html[data-theme='dark'] {
     flex-direction: row-reverse;
     justify-content: flex-end;
 
+    flex: initial;
```

Comment:
Can we add comments describing what this does?

Reply:
Removed since it seems to be no longer needed.